pkg/cloud/openstack/openstack.go

```diff
@@ -0,0 +1,48 @@
+package openstack
+
+import (
+	"bytes"
+	"embed"
+
+	appsv1 "k8s.io/api/apps/v1"
+	v1 "k8s.io/api/core/v1"
+	rbacv1 "k8s.io/api/rbac/v1"
+	k8sYaml "k8s.io/apimachinery/pkg/util/yaml"
+
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+//go:embed assets/*
+var f embed.FS
+
+func GetResources() []client.Object {
+	resources := []struct {
+		obj   client.Object
+		asset string
+	}{
+		{&v1.ConfigMap{}, "assets/config.yaml"},
+		{&appsv1.Deployment{}, "assets/deployment.yaml"},
+		{&v1.ServiceAccount{}, "assets/rbac/serviceaccount.yaml"},
+		{&rbacv1.Role{}, "assets/rbac/role.yaml"},
+		{&rbacv1.RoleBinding{}, "assets/rbac/rolebinding.yaml"},
+		{&rbacv1.ClusterRoleBinding{}, "assets/rbac/clusterrolebinding.yaml"},
+	}
+	ret := make([]client.Object, 0, len(resources))
+
+	for i := 0; i < len(resources); i++ {
+		resource := &resources[i]
+		data, err := f.ReadFile(resource.asset)
+		if err != nil {
```

Comment:
Not important, but I was wondering about it when reading embed.FS yesterday: do we need error handling here? Can this actually fail, except for an invalid path which would be picked up by the unit test.